docs/customservices.md

```diff
@@ -113,3 +113,24 @@ For Prometheus you need to set the type to Prometheus and provide a url.
   url: "http://192.168.0.151/"
   # subtitle: "Monitor data server"
 ```
+
+## AdGuard Home
+For AdGuard Home you need to set the type to AdGuard, if you have somes issues as 403 responses on requests you need to provide authentification in headers for locations needed as below.
+In `config.yml`
```

Comment:
you can remove this line, it's the same for all services.

Reply:
That's right. Done 😉